cpp/tests/utilities_tests/lists_column_wrapper_tests.cu

```diff
@@ -0,0 +1,722 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/lists/lists_column_view.hpp>
+
+#include <tests/utilities/base_fixture.hpp>
+#include <tests/utilities/column_utilities.hpp>
+#include <tests/utilities/column_wrapper.hpp>
+#include <tests/utilities/cudf_gtest.hpp>
+#include <tests/utilities/type_lists.hpp>
+
+struct ListColumnWrapperTest : public cudf::test::BaseFixture {
+};
+template <typename T>
+struct ListColumnWrapperTestTyped : public cudf::test::BaseFixture {
+  ListColumnWrapperTestTyped() {}
+
+  auto data_type() { return cudf::data_type{cudf::type_to_id<T>()}; }
+};
+
+using FixedWidthTypesNoBool = cudf::test::
+  Concat<cudf::test::IntegralTypes, cudf::test::FloatingPointTypes, cudf::test::TimestampTypes>;
+TYPED_TEST_CASE(ListColumnWrapperTestTyped, FixedWidthTypesNoBool);
```

Comment:
Can we not have a list/nested column of `BOOL8` type? Either way, there should probably be a test for it.

Reply:
You can.  I left it out because the tests in this case use values > 1 which were causes compile errors about narrowing.   I'll add some in explicitly alongside the ints/strings.

Reply:
I usually solve this by adding a `static_cast<T>(val)` wherever I access the data in a test.